solution

YevhenRDoIT opened the pull request.

YevhenDoDev committed:
solution

volodymyr-soltys97 commented on src/scripts/main.js:
```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const theadRow = table.querySelector('thead tr');
+const tbodyRows = table.querySelectorAll('tbody tr');
+const tfootRow = table.querySelector('tfoot tr');
+
+function cloneAndAppendSecondColumn(row) {
+  const secondCell = row.children[1];
+  const lastCell = row.lastElementChild;
+  const clonedCell = secondCell.cloneNode(true);
+
+  row.insertBefore(clonedCell, lastCell);
+}
+
+cloneAndAppendSecondColumn(theadRow);
+
+tbodyRows.forEach((row) => {
+  cloneAndAppendSecondColumn(row);
+});
+
+cloneAndAppendSecondColumn(tfootRow);
```
You can simplify it
```suggestion
const table = document.querySelector('table');

for (const row of table.rows) {
  const copyColumn = row.cells[1].cloneNode(true);

  row.lastElementChild.before(copyColumn);
}
```

volodymyr-soltys97 reviewed (changes requested):
Hi, there is no demo link, also, you need to add demo link to the PR description 
<img width="936" alt="image" src="https://github.com/user-attachments/assets/99fcff9c-bfb3-44d3-b2e9-5d89530cf8ca">

YevhenDoDev committed:
solution

YevhenRDoIT commented:
I have updated syntax code to the suggested one. And fixed the DEMO LINK page 
[DEMO LINK](https://YevhenRDoIT.github.io/js_task_clone_table_column_DOM/)
